src/docfx/lib/markdown/MarkdigUtility.cs

```diff
@@ -158,32 +158,39 @@ public static string ToPlainText(this MarkdownObject containerBlock)
 
         public static bool IsVisible(this MarkdownObject markdownObject)
         {
-            var visiable = false;
+            var visible = false;
             Visit(markdownObject, node =>
             {
                 switch (node)
                 {
                     case HtmlBlock htmlBlock:
                         foreach (var line in htmlBlock.Lines.Lines)
                         {
-                            visiable = visiable || VisibleHtml(line.Slice.Text);
+                            visible = visible || VisibleHtml(line.Slice.Text);
```

Comment:
would `visible |= VisibleHtml(line.Slice.Text);` just work?

Reply:
both way should be OK?